app/routes/users/components/UserProfile.js

```diff
@@ -57,6 +57,7 @@ type Props = {
   feed: Object,
   isMe: boolean,
   loading: boolean,
+  previousEvents: Array<Event>,
   upcomingEvents: Array<Event>,
```

Comment:
Should these two use the types defined below? And also maybe merge the two types into 1?